src/assets/translations/fr/main.json

```
@@ -753,11 +758,14 @@
     "customRecipientAddressDescription": "Saisissez une adresse de destinataire personnalisée pour cet échange",
     "thisIsYourCustomRecipientAddress": "Il s'agit de l'adresse personnalisée du destinataire",
     "enterCustomRecipientAddress": "Saisissez l'adresse de destinataire personnalisée",
+    "receiveAtLeast": "Vous recevez au moins",
```

Would prefer one of these two to match the imperative tense in English
```suggestion
    "receiveAtLeast": "Vous recevrez au moins",
```
```suggestion
    "receiveAtLeast": "Recevez au moins",
```